### PR TITLE
upgrade druid to 0.9.2 and add swap space to Avoid OOME

### PR DESCRIPTION
**Pull request by nishantmonu51**
This PR has 2 changes - 
1) Update druid version to 0.9.2 
2) Add swap space to avoid OOME errors, NOTE: this slows down the tests a bit. On my machine it now takes about 13 minutes for all druid tests to run. 
Alternatively, we can increase the RAM allocated to the VM to make the tests faster.

### CODE REVIEW

File: foodmart-dataset/src/main/resources/druid/install.sh [hunk 1]
```diff
@@ -32,14 +33,14 @@ echo Install and start Zookeeper
 )
 
 echo Removing previous Druid
-rm -rf druid-0.9.1.1
+rm -rf druid-*
 
 echo Installing Druid
-if [ ! -f /var/cache/apt/archives/druid-0.9.1.1-bin.tar.gz ]; then
-  curl --silent http://static.druid.io/artifacts/releases/druid-0.9.1.1-bin.tar.gz -o /var/cache/apt/archives/druid-0.9.1.1-bin.tar.gz
+if [ ! -f /var/cache/apt/archives/druid-${druid_version}-bin.tar.gz ]; then
```
Comment (b-slim) [inline foodmart-dataset/src/main/resources/druid/install.sh]: minor can we print the version installed here ?

**Comment by b-slim** [comment]
👍  Thanks Nishant i guess we can also tweak the conf to use less ram via jvm args but we can do that in a followup PR.

File: foodmart-dataset/src/main/resources/druid/install.sh (hunk 1 above)
Comment (nishantmonu51) [reply foodmart-dataset/src/main/resources/druid/install.sh]: done.

**Comment by julianhyde** [comment]
See discussion in https://issues.apache.org/jira/browse/CALCITE-1730.

**Comment by julianhyde** [comment]
+1 Calcite suite passes (with fixes for https://issues.apache.org/jira/browse/CALCITE-1734 applied).

@vlsi Can you commit? Please add a header to `create_swap.sh` if @nishantmonu51 doesn't do it first.

**Pull request merged**